src/community/release/pom.xml

```diff
@@ -440,4 +403,74 @@
    </plugin>
   </plugins>
  </build>
+ 
+ <!-- profiles for individual modules -->
+ <profiles>
+   <profile>
+     <id>rest-ext</id>
+     <activation>
+       <property><name>rest-ext</name></property>
+     </activation>
+     <dependencies>
+       <dependency>
+         <groupId>org.geoserver.community</groupId>
+         <artifactId>gs-rest-ext</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+     </dependencies>
+   </profile>
+   <profile>
+     <id>script</id>
+     <activation>
+       <property><name>script</name></property>
+     </activation>
+     <dependencies>
+       <dependency>
+         <groupId>org.geoserver.script</groupId>
+         <artifactId>gs-script-py</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+       <dependency>
+         <groupId>org.geoserver.script</groupId>
+         <artifactId>gs-script-js</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+       <dependency>
+         <groupId>org.geoserver.script</groupId>
+         <artifactId>gs-script-groovy</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+     </dependencies>
+   </profile>
+   <profile>
+     <id>monitor-hibernate</id>
+     <dependencies>
+       <dependency>
+        <groupId>org.geoserver.community</groupId>
+        <artifactId>gs-monitor-hibernate</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+     </dependencies>
+   </profile>
+   <profile>
+     <id>geogig</id>
+     <dependencies>
+       <dependency>
+         <groupId>org.geoserver.community</groupId>
+         <artifactId>gs-geogig</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+    </dependencies>
+   </profile>
+   <profile>
+     <id>rest-upload</id>
+     <dependencies>
+       <dependency>
+         <groupId>org.geoserver.community</groupId>
+         <artifactId>gs-rest-upload</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+     </dependencies>
+   </profile>
+ </profiles>
```

Comment:
This looks to be close to what is neccessary for individual assembly, but does not appear to be the same number of modules that you added release descriptors to in the community pom; spatialite at least appears to be missing (so assembly would fail)

Reply:
As you noted the codebase was in consistent previously.

gs-spatialite was not mentioned anywhere here before, I can add it back as a profile, if only as a starting point for anyone wishing to restore it.